/*
 * controller_core.h
 *
 *  Created on: May 5, 2012
 *      Author: Michael Anderson
 */

#ifndef CONTROLLER_CORE_H_
#define CONTROLLER_CORE_H_

#include <ac_leg_force/controller_common.h>
#include <atrias_control/core_library.h>
#include <drl_library/drl_math.h>
#include <math.h>

/*float springDeflectionA[250];
float springDeflectionB[250];
float velABuf[100];
float velBBuf[100];
int velBufLoc;
int ringBufferLocation;*/
uint16_t elapsed_time;
bool chirp_on;

#endif /* CONTROLLER_CORE_H_ */

